README.md

```diff
@@ -1,9 +0,0 @@
-# trackitdown2020Analytics
-SEIR model in Python for React App trackitdown2020 (https://github.com/trackitdown2020)
```

Comment:
Would recommend you actually add the documentation including 
(1) How to get this up and running with venv 
(2) The endpoint that we will be hitting locally and in heroku 
(3) A general overview of what's happening behind the scenes. 
1 and 2 are must haves, 3 is a up to your discretion how detailed but it'll be nice to document for the next person.